test/fuzzer.cpp

```diff
@@ -49,6 +53,33 @@ int main()
 		input += s + '\n';
 	}
 
+	if (standard)
+	{
+		string outputString(compileStandard(input.c_str(), NULL));
+		Json::Value output;
+		if (!Json::Reader().parse(outputString, output))
+		{
+			cout << "Compiler produced invalid JSON output." << endl;
+			abort();
+		}
+		if (output.isMember("errors"))
+			for (auto const& error: output["errors"])
+			{
+				string invalid = contains(error["type"].asString(), vector<string>{
+					"JSONError",
```

Comment:
I don't think it would *only*, but definitely would find them. Having checked it again, I use `JSONError` to signal a field is missing or JSON cannot be parsed (so that is a safe exit), while actual logic errors of reading JSON is within `InternalCompilerError`. It should be fine to remove `JSONError` in that case.